chia/cmds/dev.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import annotations
+
+import click
+
+from chia.cmds.sim import sim_cmd
+
+
+@click.group("dev", short_help="Developer commands and tools")
```

Comment:
Yeah, what you've got here is consistent, but I'm in the middle of redefining the consistency.  https://github.com/Chia-Network/chia-blockchain/pull/15076  This applies to any other places you don't specify a more complete `help=`.

```suggestion
@click.group("dev", help="Developer commands and tools")
```